### PR TITLE
Use 'as name' patter to re-export names from stubs (PEP 484)

### PR DESCRIPTION
**Pull request by ilevkivskyi**
These were found with the help of https://github.com/python/mypy/pull/3706/

I am a bit worried that there might be more errors like these, but they are not caught, since some names are not imported (e.g. in top-level modules).

@JelleZijlstra please take a look. This should be merged (if OK) before https://github.com/python/mypy/pull/3706/

### CODE REVIEW

**Comment by gvanrossum** [comment]
Please hold off on this until after the release of 0.521 (or until Jukka vets it).

File: stdlib/3.4/asyncio/futures.pyi [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import sys
-from typing import Any, Union, Callable, TypeVar, List, Generic, Iterable, Generator, Awaitable
+from typing import Any, Union, Callable, TypeVar, List, Generic, Iterable, Generator, Awaitable as Awaitable
```
Comment (JelleZijlstra) [inline stdlib/3.4/asyncio/futures.pyi]: Why is this here? Awaitable doesn't exist in this module at runtime.

File: third_party/2and3/requests/packages/urllib3/__init__.pyi [hunk 2]
```diff
@@ -1,8 +1,8 @@
 from typing import Any
 from . import connectionpool
-from . import filepost
-from . import poolmanager
-from . import response
+from . import filepost as filepost
```
Comment (JelleZijlstra) [inline third_party/2and3/requests/packages/urllib3/__init__.pyi]: Is this really needed? That feels like a bug in mypy or the PEP; child modules should always be re-exported.

File: stdlib/3.4/asyncio/futures.pyi (hunk 1 above)
Comment (ilevkivskyi) [inline stdlib/3.4/asyncio/futures.pyi]: Yes, it look like this is a bug at point of import in ``asyncio/locks.pyi``

File: third_party/2and3/requests/packages/urllib3/__init__.pyi (hunk 2 above)
Comment (ilevkivskyi) [inline third_party/2and3/requests/packages/urllib3/__init__.pyi]: This is kind of a corner case. PEP is very brief here saying just "everything that does not use ``as name`` is hidden (except for ``import *``)". But probably this is indeed too verbose, I can special case this in mypy. Also probably some clarification in the PEP (with examples) is needed. Would you like making a PR to the ``python/peps`` repo?

Comment (gvanrossum) [inline third_party/2and3/requests/packages/urllib3/__init__.pyi]: Oh, good catch. This is covered by a different notion, where submodules of packages are made available as attributes of their parent package as a side-effect of importing the submodule (implicitly or explicitly). So the 'as' idiom should not be needed here. But yeah, a PEP clarification would be good.

Comment (ilevkivskyi) [inline third_party/2and3/requests/packages/urllib3/__init__.pyi]: OK, I updated this PR and the mypy PR accordingly (the 'as name' rule does not apply to child modules now).

File: third_party/2and3/jinja2/utils.pyi [hunk 3]
```diff
@@ -1,6 +1,6 @@
 from typing import Any, Optional
 
-from markupsafe import Markup, escape, soft_unicode
+from markupsafe import Markup as Markup, escape as escape, soft_unicode as soft_unicode
```
Comment (JelleZijlstra) [inline third_party/2and3/jinja2/utils.pyi]: This is a little strange (Jinja2 exports functions from markupsafe directly), but confirmed that it's correct.

**Comment by JelleZijlstra** [comment]
Thanks!
